e2e_test/source/cdc_inline/auto_schema_map_mysql.slt

```diff
@@ -0,0 +1,118 @@
+control substitution on
+
+# test case need to cover all data types
+system ok
+mysql --protocol=tcp -u root -e "DROP DATABASE IF EXISTS mytest; CREATE DATABASE mytest;"
```

Comment:
I think we can omit `--protocol=tcp` and `-u root` and directly use `mysql -e` in `.slt`.

https://github.com/risingwavelabs/risingwave/blob/d19d8f9ae55832430f452c2ed90fca44910dd37b/e2e_test/source_inline/cdc/mysql/mysql_create_drop.slt#L8-L9